app/controllers/api/integrations_controller.rb

```diff
@@ -13,6 +13,11 @@ class Api::IntegrationsController < Api::BaseController
   rescue_from ActiveRecord::StaleObjectError, with: :edit_stale
 
   def edit
+    begin
+      @registry_policies = Policies::PoliciesListService.call!(current_account)
+    rescue RuntimeError, Errno::ECONNREFUSED => error
```

Comment:
You can use `rescue_from`

Reply:
I like what Hery is suggesting but that changes a lot the scope of the task. So we should probably create another Jira issue for doing that